Dockerfile

```diff
@@ -63,29 +64,30 @@ COPY ${NETBOX_PATH} /opt/netbox
 COPY docker/configuration.docker.py /opt/netbox/netbox/netbox/configuration.py
 COPY docker/docker-entrypoint.sh /opt/netbox/docker-entrypoint.sh
 COPY docker/launch-netbox.sh /opt/netbox/launch-netbox.sh
+COPY docker/housekeeping.sh /opt/netbox/housekeeping.sh
 COPY startup_scripts/ /opt/netbox/startup_scripts/
 COPY initializers/ /opt/netbox/initializers/
 COPY configuration/ /etc/netbox/config/
 COPY docker/nginx-unit.json /etc/unit/
 
-WORKDIR /opt/netbox/netbox
+WORKDIR /opt/netbox/
 
 # Must set permissions for '/opt/netbox/netbox/media' directory
 # to g+w so that pictures can be uploaded to netbox.
 RUN mkdir -p static /opt/unit/state/ /opt/unit/tmp/ \
-      && chmod -R g+w media /opt/unit/ \
-      && cd /opt/netbox/ && /opt/netbox/venv/bin/python -m mkdocs build \
+      && chmod -R g+w /opt/netbox/netbox/media /opt/unit/ \
+      && /opt/netbox/venv/bin/python -m mkdocs build \
           --config-file /opt/netbox/mkdocs.yml --site-dir /opt/netbox/netbox/project-static/docs/ \
       && SECRET_KEY="dummy" /opt/netbox/venv/bin/python /opt/netbox/netbox/manage.py collectstatic --no-input
 
-ENTRYPOINT [ "/opt/netbox/docker-entrypoint.sh" ]
+ENTRYPOINT [ "/sbin/tini", "--" ]
```

Comment:
Why is an init necessary to achieve the houskeeping? Isn't the housekeeping just a script that pauses for 24h?

Reply:
This is also a breaking change, and I would like to avoid it. Because it changes the "user interface" to the Docker Image and therefore adjustments to someone's custom docker-compose file or Kubernetes config might be necessary if they rely on the entrypoint being to what it 'always' was.

Reply:
The init is not strictly necessary for the housekeeping script, but is recommend. But you are right, this is a breaking change.
So it should be done in a major version. Probably better to split it from the introduction of the housekeeping script.